docs/spl-docs/search-command.md

```diff
@@ -0,0 +1,91 @@
+# search command
+
+## Description
+
+The search command is a versatile tool that allows you to retrieve events from your indexes and filter the results of previous search commands in your pipeline. 
+You can use keywords, quoted phrases, wildcards, and field-value expressions to specify the events you want to retrieve from your indexes. 
+The search command is implied at the beginning of any search, so you don't need to explicitly include it when defining your search criteria.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+search **\<search-expression\>**
+
+
+### Required Arguments
+
+#### \<search-expression\>
+
+**Syntax:** `<literal-expression> | <comparison-expression>`\
```

Comment:
nit: since `|` is used in SPL, I think something like this will be more clear:
```
**Syntax:** ether `<literal-expression>` or `<comparison-expression>`\
```

Same comment for some other places

Reply:
wouldn't it be too cluttered for some commands like bin, where timescale has many options?

Reply:
I don't think so. Something like:
> either `foo`, `bar`, `baz`, or `fizz`

is slightly easier to understand and not much more cluttered than
> `foo | bar | baz | fizz`

Also, normally I would agree that `foo | bar | baz | fizz` is good; it's just that since SPL uses `|` to pipe commands, using it in this context could be a little confusing to readers

Reply:
Also if it does get too cluttered for some commands, you can use your judgement to make it more organized (e.g., list them as bullet points)

Reply:
addressed in the latest commit.